src/Module/Moderation/Summary.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2022, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\Moderation;
+
+use Friendica\Core\Logger;
+use Friendica\Core\Renderer;
+use Friendica\Database\DBA;
+use Friendica\DI;
+use Friendica\Model\Register;
+use Friendica\Module\BaseModeration;
+
+class Summary extends BaseModeration
+{
+	protected function content(array $request = []): string
+	{
+		parent::content();
```

Comment:
You can delete this since it's just a empty hull

Reply:
What is an empty hull?

Reply:
Actually this call is checking credentials, so far from being an empty hull!

Reply:
oups .. thanks for the hint @MrPetovan ..